client/src/common/components/Form.jsx

```diff
@@ -74,6 +75,11 @@ class Form extends PureComponent {
             type="text"
             value={name}
           />
+          {errorMessageId && (
+            <span className="error-message">
+              <FormattedMessage id="common.form.default-warning" />
```

Comment:
I think the `errorMessageId` should be used, otherwise, if there is an unknown error - only then it should fall back to the default error message.